javalib/src/main/scala/java/lang/String.scala

```diff
@@ -796,8 +796,8 @@ object _String {
   }
 
   import scala.language.implicitConversions
-  @inline private implicit def _string2string(s: _String): String =
+  @inline private[java] implicit def _string2string(s: _String): String =
     s.asInstanceOf[String]
-  @inline private implicit def string2_string(s: String): _String =
+  @inline private[java] implicit def string2_string(s: String): _String =
```

Comment:
Why do you need making those public? You can just keep using normal `String` type everywhere outside of `java/lang/String.scala` and it would map to the same class at runtime automatically.